src/ingest-pipeline/misc/tools/new_dataset_survey.py

```diff
@@ -249,11 +365,8 @@ def main():
     out_df = out_df.sort_values([key for key in ROW_SORT_KEYS if key in out_df.columns],
                                 axis=0)
 
-    out_df.to_csv(args.out, sep='\t', index=False,
```

Comment:
I propose that there be a command line option to send directly to Google Sheets, so the old functionality is available if needed.

Reply:
Different parameters to main?  I'm not sure I follow what you're proposing.  I thought to just add a command line flag like --to_csv to cause the old behavior to occur.  (I presume one can still add additional --notes from the command line)

Reply:
Well, the behavior and parameter did change, now you don't use paths for it (--out and --notes). --out now uses the name of the new sheet you want and --notes was updated to --sheet_name (which in retrospective isn't a good name for it). So, my counter-proposal is to rename parameters so they are more adequate to the function you require like this. --to_csv --out_file --notes_file and --out_sheet --notes_sheet (for the new functionality)

Reply:
That makes perfect sense- I like those names.